### PR TITLE
Added a start screen to the game, made it look cool, and made PixeloidMono the default font for everything in CSS.

### PR DESCRIPTION
**Pull request by JudeKlatte**
START SCREEEEEEEEEEEN

### CODE REVIEW

File: frontend/public/style.css [hunk 1]
```diff
@@ -1,3 +1,29 @@
+#start-screen {
+  background-color: #000000;
+  color: #66ff33;
+  text-align: center;
+  font-size: 40px;
+  justify-content: center;
+  /*align-items: center;*/
```
Comment (jvgusch) [inline frontend/public/style.css:7]: Recommend removing unused code.

**Comment by emadsu** [comment]
the "play" button area is a little big, otherwise amazing and extremely true to the mock-up

File: frontend/src/main.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import store from "./store.js";
 import { Provider } from "react-redux";
 import Profile from "./components/Auth/Profile.js";
 import PrivateRoute from "./components/Auth/PrivateRoute.js";
+import StartScreen from "./StartScreen";
```
Comment (jvgusch) [inline frontend/src/main.tsx:10]: Unused import.

**Review by jvgusch — APPROVED** [review]
Other than some cleanup I'd recommend, I think this is great! Nice work!

File: frontend/public/style.css (hunk 1 above)
Comment (JudeKlatte) [reply frontend/public/style.css]: I removed it in my second commit change

**Review by emadsu — APPROVED** [review]
what a king

**Review by josephmaxwellhyland — APPROVED** [review]
Looks good to me!

**Review by jvgusch — APPROVED** [review]

**Pull request merged**